test/v2/models/request/common/ukFhlPieProperty/UkFhlPiePropertyIncomeSpec.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.request.common.ukFhlPieProperty
+
+import play.api.libs.json.{JsValue, Json}
+import support.UnitSpec
+import v2.models.request.common.ukPropertyRentARoom.UkPropertyIncomeRentARoom
+
+
+class UkFhlPiePropertyIncomeSpec extends UnitSpec{
+   val rentARoom: UkPropertyIncomeRentARoom= UkPropertyIncomeRentARoom(Some(412.89))
+   val requestBody: UkFhlPiePropertyIncome = UkFhlPiePropertyIncome(Some(215.16), Some(1365.12), Some(rentARoom))
+
+  val mtdJson: JsValue = Json.parse(
+    """
+      |{
+      |    "periodAmount": 215.16,
+      |    "taxDeducted": 1365.12,
+      |    "rentARoom": {
+      |      "rentsReceived": 412.89
+      |   }
+      |}
+      |""".stripMargin)
+
+  val desJson: JsValue = Json.parse(
+    """
+    |{
+    |    "rentIncome": {
+    |    "amount": 215.16,
+    |    "taxDeducted": 1365.12
```

Comment:
Could this json be formatted a bit more clearly?

Reply:
absolutely